engine/src/main/scala/cromwell/server/CromwellSystem.scala

```diff
@@ -13,9 +15,13 @@ trait CromwellSystem {
   val conf = ConfigFactory.load()
   val logger = LoggerFactory.getLogger(getClass.getName)
   implicit final lazy val actorSystem = newActorSystem()
+  implicit final lazy val materializer = ActorMaterializer()
+  implicit private final lazy val ec = actorSystem.dispatcher
 
   def shutdownActorSystem(): Future[Terminated] = {
-    actorSystem.terminate()
+    Http().shutdownAllConnectionPools() map { _ =>
+      materializer.shutdown()
+    } flatMap { _ => actorSystem.terminate() }
```

Comment:
IIRC (which i probably don't as I didn't go look) this is actually what the docs show to do